Oracle/diagnosis.sql

```diff
@@ -124,8 +124,8 @@ execute immediate 'truncate table pdxfact';
 execute immediate 'truncate table originfact';
 execute immediate 'truncate table poafact';
 
-insert into sourcefact
-	select distinct patient_num, encounter_num, provider_id, concept_cd, start_date, dxsource.pcori_basecode dxsource, dxsource.c_fullname
+INSERT /*+ APPEND */ into sourcefact
+	select /*+ parallel */ distinct patient_num, encounter_num, provider_id, concept_cd, start_date, dxsource.pcori_basecode dxsource, dxsource.c_fullname
```

Comment:
Does parallel without a number work reasonably well in your experience, @contactlp ? Is Oracle smart enough to pick a reasonable number?

I remember at least one time when Oracle went 128x parallel because it saw 128 cores on yellowfin or something like that. And it slowed to a crawl. Remember, @dbudinekumc ?